prep-html-syntax/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <h1 style="color: blue;">My favorite Star Wars character is yoda.</h1>
+    <img src="https://static.wikia.nocookie.net/starwars/images/d/d6/Yoda_SWSB.png/revision/latest?cb=20150206140125"
+      alt="yoda">
+      <p style="color: green;">Yoda is my favorite because he is wise.</p>
+  </body>
+</html>
```

Comment:
Hi Jun, be sure to divide your page into two specific sections. The `<head>` should open and close before the `<body>` opens and closes. The visible content of the page should go in the `<body>`.

```html
<!DOCTYPE html>
<html>

  <head>
    <!-- meta info like <title></title> goes in here -->
  </head>

  <body>
    <!-- visible content like <h1></h1> and <p></p> goes in here -->
  </body>

</html>
```

Reply:
![Screenshot 2021-08-02 090135](https://user-images.githubusercontent.com/88108910/127890768-1b78219b-392c-497f-8654-2c745da8e19d.png)
![Screenshot 2021-08-02 090210](https://user-images.githubusercontent.com/88108910/127890805-0ad8e2e9-7b40-48f8-99b2-5e7e8552dfa9.png)
![updated ](https://user-images.githubusercontent.com/88108910/127890956-bdbcd3f4-b6d5-4ed5-b4e2-ae586fdb6f22.png)